src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/spreadsheet/PriceSheet.kt

```diff
@@ -107,8 +194,8 @@ abstract class PriceSheet(val sheet: Sheet, private val header: Header) {
       add(9, vehicleRegistration)
       add(10, person?.prisonNumber)
       if (hasPrice()) row.addCell(11, totalInPounds(), fillShadedPound) else add(11, "NOT PRESENT")
-      add(12, "") // billable is empty for a move
-      add(13, if (showNotes) notes else "")
+      if (dataColumns.contains(DataColumn.CONTRACTOR_BILLABLE)) add(12, "") // billable is empty for a move
```

Comment:
We only add the columns if they are part of the collections of columns of the current sheet/tab.